README.md

```diff
@@ -1,2 +1,154 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Books: JS objects <a name="about-project"></a>
+
+**Awesome books** is a simple website that displays a list of books and allows you to add and remove books from that list. By building this application, I've learnt how to manage data using JavaScript. That's why the website will be more interactive. I used the given medium-fidelity wireframe to build the UI.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">index.html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Javascript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">script.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.css</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add books to the book list**
+- **Remove any book in the list**
+- **Create your own book list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Awesome Books: JS objects](https://nedramrez.github.io/awesome-books/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Open VSCode and type <code>ctrl+P</code>, type
+
+```sh
+  ext install ritwickdey.liveserver
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone git@github.com:Nedramrez/awesome-books.git;
+```
+
+### Usage
+
+To run the project just click the Go Live button in the VSCode.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zermarden**
+
+- GitHub: [@zermarden](https://github.com/Nedramrez)
+- Twitter: [@zermarden](https://twitter.com/zermarden)
+- LinkedIn: [@zermarden](https://linkedin.com/in/zermarden)
+
+👤 **Solomon Opeyemi**
+
+- GitHub: [@solorics](https://github.com/solorics)
+- Twitter: [@twitterhandle](https://twitter.com/solorics)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/solorics)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```

Comment:
- [ ] [optional] @team great work on your project, it would have been nice if you added future features you intend to implement in this project.